lightning/src/util/logger.rs

```diff
@@ -29,6 +31,7 @@ static LOG_LEVEL_NAMES: [&'static str; 6] = ["GOSSIP", "TRACE", "DEBUG", "INFO",
 
 /// An enum representing the available verbosity levels of the logger.
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
Correct. It's to avoid writing stuff like [this](https://github.com/tnull/ldk-node-hack-server/blob/33a1bd8c336f1ad69931fd541143ec3f6cdef38f/server/src/config.rs#L22)